server/src/services/store/templates/draft-2-command-line-tool.hbs

```diff
@@ -0,0 +1,45 @@
+{
+    "id": "{{ id }}",
+    "class": "CommandLineTool",
+    "label": "{{ label }}",
+    "description": "{{ description }}",
```

Comment:
All the spec says about the id is literally `The unique identifier for this process object.` It isn't even a required field. The ideal id for local development might just be the absolute path of the file.

It is important to fill out the description, it should be removed from the new file form and be given its own dedicated space in the UI. Moreover, the practice thus far has been that the description can handle Markdown, so it ought to have a fancy editor.